.github/workflows/publish.yml

```diff
@@ -44,7 +44,7 @@ jobs:
              # constraining environment for riscv64 builds
              TAG=lfedge/eve-alpine:81e6520f4f1554789eb1ac299168e72ac37d88e2-riscv64
              echo "ZARCH=riscv64" >> "$GITHUB_ENV"
-             echo 'LK_BUILD_ARGS={"EVE_BUILDER_IMAGE":"'$TAG\"} >> "$GITHUB_ENV"
+             echo 'LK_BUILD_ARGS={"ALPINE_VERSION":"3.13","EVE_BUILDER_IMAGE":"'$TAG\"} >> "$GITHUB_ENV"
```

Comment:
Why 3.13 here?

Reply:
riscv64 packages was not ready, but now they are, so I updated PR to build riscv64 EVE as well on top of Alpine 3.16